docs/Setup/WSL-Guide.md

```diff
@@ -91,3 +91,48 @@ Note that currently, TBA is porting a significant portion of code from Python 2
 ### Run `vagrant up`
 
 Try running `vagrant up`. If you see an error regarding unable to find the directory `/tba`, try running `vagrant destroy && vagrant up`.
+
+
+### Troubleshooting rsync
+
+You may find that rsync is especially slow on WSL; currently, I/O speed is a problem for WSL. A simple workaround is to use [vagrant-gatling-rsync](https://github.com/smerrill/vagrant-gatling-rsync).
+
+Install with the following command:
+
+```
+$ vagrant plugin install vagrant-gatling-rsync
+```
+
+Modify `Vagrantfile` to contain the following:
```

Comment:
Is there a way to do this programmatically vs. making the edit manually?

Reply:
Yeah, done, I added `ops/dev/vagrant/fix_wsl_rsync.sh` that runs a couple `sed` commands to modify local files.